machine_learning/mfcc.py

```diff
@@ -209,7 +209,9 @@ def calculate_fft(audio_windowed: np.ndarray, ftt_size: int = 1024) -> np.ndarra
         >>> import numpy as np
         >>> audio_windowed = np.array([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]])
         >>> audio_fft = calculate_fft(audio_windowed, ftt_size=4)
-        >>> np.allclose(audio_fft, np.array([[6.0+0.j, -1.5+0.8660254j, -1.5-0.8660254j], [15.0+0.j, -1.5+0.8660254j, -1.5-0.8660254j]]))
+        >>> np.allclose(\
```

Comment:
PEP8: Avoid backslash line termination in Python code.
```suggestion
        >>> np.allclose(
```

Reply:
Thanks for the feedback. When I remove the backslashes the doctest will fail to parse the test example. When I put all of them in the same line the ruff will raise an error that the line is too long.

Reply:
Read PEP8.  Backslashes are not required inside (), [], {}.

Reply:
Done.